MoonDawgArduino/Libraries/Movement/Movement.cpp

```diff
@@ -12,64 +12,63 @@ DigMovement::DigMovement() {
     // Empty constructor
 }
 
-void DigMovement::digSetup(uint8_t diggingPin, uint8_t actuatorPinOne, uint8_t actuatorPinTwo) {
-    pinOneAct = actuatorPinOne;
-    pinTwoAct = actuatorPinTwo;
-    pinMode(pinOneAct, OUTPUT);
-    pinMode(pinTwoAct, OUTPUT);
-    digitalWrite(pinOneAct, LOW);
-    digitalWrite(pinTwoAct, LOW);
-    motor.attach(diggingPin);
-    motor.write(90);
-}
-
-void DigMovement::digMotorForward(uint8_t maxSpeed){
-    motor.write(180);
+void DigMovement::digSetup(uint8_t diggingPin, uint8_t actuatorPin) {
+    motor1.attach(diggingPin);
+    motor2.attach(actuatorPin);
+    motor1.write(90);
+    motor2.write(90);
 }
 
-void DigMovement::digMotorBackward(){
-    motor.write(0);
+void DigMovement::digMotor(uint8_t speed){

```

Comment:
maybe call this something more specific, like digMotorStart()

Reply:
action words for functions >>

Reply:
done